.editorconfig

```diff
@@ -6,19 +6,20 @@ root = true
 
 # Default settings:
 # A newline ending every file
-# Use 2 spaces as indentation
-# Trim trailing whitespace
+# Use 4 spaces as indentation
 [*]
 insert_final_newline = true
```

Comment:
I think the place we are moving towards is to have the file have a final newline when it's human generated and just accept whatever tool generated files generate.

I don't have a strong preference either way, but Git and GitHub seem to prefer having the additional newline, so I'm fine with adding it when it doesn't require gymnastics.